src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
+}
```

Comment:
It's not recommended to use public static fields for storing data as it exposes the internal representation of the Storage. Instead, consider encapsulating the storage mechanism and providing methods to interact with it.

Reply:
++